enhancement/2007 - Add more GA aircraft

jaialin opened the pull request:
Resolves #2007 

Added

Props
- BE10
- BE20
- BE33
- BE35
- BE56
- C414
- C421
- DA62
- P28A
- P28T
- PA31
- PA44
- PC6T
- PC12
- PC24
- SR20
- SR22
- SF50

Jets

- C25A
- C25B
- C25C
- C525
- C56X
- C700
- FA20
- FA5X
- FA7X
- FA8X
- GL7T
- GLF2
- GLF3
- PC24

Amended n.json to include aircraft, and to include new mediumGA category

jaialin committed:
This pull request refines KCLT.json to conform more closely to real-world procedures.

1. Added 2DME waypoints off RWYS 18L, 18C, 18R, where planes turn onto Noise Abatement heading
2. Amended south departures to fly Noise abatement headings (this also resolves conflicts alerts during parallel departures)
3. Fixed typo on STOCR arrival which prevented arrivals from being assigned another runway
4. added FedEx to FILPZ3 arrivals (to emulate flights from Memphis and Louisville)
5. Removed UPS from SIDS/STARS they don't use in real life, adjusted spawn ratio to 0.1
6. removed DLH from SIDS STARS that they don't use (they only arrive on CHSLY5 and depart on BARMY4.RDU)
7. added FastGA planes to most arrivals

jaialin committed:
Adding Cirrus & Pilatus ACFT

SR22, SF50, PC12

added Gulfstream jets into fastGA list

jaialin committed:
Amended n.josn to include more GA acft

fixed some typos

jaialin committed:
Update n.json

fastGA is now jets only
added TurboGA for turboprops

erikquinn commented:
@jaialin soooo is this done then, or?

If it is, please request a review from the airport reviewers team.

jaialin committed:
Merge branch 'develop' into enhancement/2007

erikquinn commented:
@jaialin anything with the `airport/airline/aircraft` label just for the airport-reviewers team please 🙂 I've updated accordingly already!

They will get to it when they can.

Fechulo commented on assets/aircraft/pc12.json:
```diff
@@ -34,4 +34,4 @@
         "ils": true,
         "fix": true
     }
-}
```
Add new line at the end of file

Fechulo replied:
There is still a missing new line

Fechulo reviewed (changes requested):
Looks like some changes to KCLT from #2001 sneaked into the branch. Please remove them from this PR.

jaialin committed:
Delete kclt.json

jaialin committed:
Revert "Delete kclt.json"

This reverts commit abdb4225c6e4611c5391ce2a6fd3c077db9c5563.

jaialin committed:
Update kclt.json

jaialin committed:
Update pc12.json

jaialin committed:
Even more GA aircraft + tweaks

Added BE10, BE33, BE35, BE56, C25a, C25b, C25c, C414, C421, C525, C56X, C700, FA20, FA5X, FA7X, FA8X, GL7T, GLF2, GLF3, P28R, P28T, PA31, PA34, PA44

tweaks to performance figures of many GA aircraft

modified N.json to include all new airplanes. added new category of medium GA aircraft to include twin's and mono-turbos.

jaialin committed:
a few more GA, and update to n.json

DA62

added more aircraft to n.json list

jaialin committed:
Update n.json
